pkg/drivers/linode/linode.go

```diff
@@ -216,8 +252,32 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 func (d *Driver) PreCreateCheck() error {
 	// TODO linode-stackscript-file should be read and uploaded (private), then used for boot.
 	// RevNote could be sha256 of file so the file can be referenced instead of reuploaded.
-	// linode-stackscript would let the user specify an existing id
-	// linode-stackscript-data would need to be a json input
+
+	if d.StackScriptUser != "" {
+		client := d.getClient()
+
+		options := map[string]string {
+			"username": d.StackScriptUser,
+			"label": d.StackScriptLabel,
+		}
+		b, err := json.Marshal(options)
+		if err != nil {
+			return err
+		}
+		opts := linodego.NewListOptions(0, string(b))
+		stackscripts, err := client.ListStackscripts(context.TODO(), opts)
+		if err != nil {
+			return err
+		}
+		if len(stackscripts) != 1 {
+			return fmt.Errorf("StackScript %s/%s not found", d.StackScriptUser, d.StackScriptLabel)
+		}
+
+		d.StackScriptID = stackscripts[0].ID
+
+		log.Infof("Using StackScript %s/%s (%d)", d.StackScriptUser, d.StackScriptLabel, d.StackScriptID)
```

Comment:
`%s/%s` should be single-quoted, as the label can contain non-alphanumeric characters:
`Using StackScript displague/Rootless and GitHub User has sudo (10079)`

But it could be even more confusing:
`Using StackScript displague/Rootless & GitHub/User.keys installed - copy (2) (10079)`

Reply:
Possibly, yes. I included it because I wanted it to be obvious (a) that the script was running and (b) which script was running. I expect it'll be easy enough to escape it somehow (I'm not really a Go programmer — today is the first time I've really bothered with it at all).